includes/class-amp-story-post-type.php

```diff
@@ -1871,7 +1875,7 @@ public static function render_block_page_attachment( $attributes ) {
 		?>
 		<amp-story-page-attachment layout="nodisplay" theme="light" data-cta-text="<?php echo esc_attr( $attributes['text'] ); ?>" data-title="<?php echo esc_attr( $attributes['title'] ); ?>">
 			<div class="<?php echo esc_attr( $attributes['attachmentClass'] ); ?>" style="<?php echo esc_attr( $style ); ?>">
-				<h2><?php echo esc_html( $content_post->post_title ); ?></h2>
+				<h2><?php echo the_title(); ?></h2>
```

Comment:
echo not required here.

Reply:
This just crossed with my fix in 5e47558c097059e00eb145586a1595c3ee16899f, so consider it resolved 🙂